app/graph/types/version_type.rb

```diff
@@ -1,61 +1,56 @@
 VersionType = GraphqlCrudOperations.define_default_type do
   name 'Version'
-  description 'Version type'
+  description 'An entry in the version-control log.'
 
   interfaces [NodeIdentification.interface]
 
-  field :dbid, types.Int
-  field :item_type, types.String
-  field :item_id, types.String
-  field :event, types.String
-  field :event_type, types.String
-  field :object_after, types.String
-  field :meta, types.String
-  field :object_changes_json, types.String
-  field :associated_graphql_id, types.String
-  field :smooch_user_slack_channel_url, types.String
-
-  field :user do
-    type -> { UserType }
-
+  field :item_type, types.String, 'Database type of item' # TODO Consider enum type https://graphql.org/learn/schema/#enumeration-types
+  field :item_id, types.String, 'Database id of item'
+  field :event, types.String # TODO Document
+  field :event_type, types.String # TODO Document
+  field :object_after, types.String # Do we need both this and 'object_changes'?
+  field :meta, types.String # TODO Document
+  field :object_changes_json, types.String # TODO Convert to JsonStringType and rename to 'item_diff'
+  field :associated_graphql_id, types.String # TODO Document
+  field :smooch_user_slack_channel_url, types.String # TODO Document
+
+  field :user, -> { UserType }, 'Log entry creator' do
     resolve ->(version, _args, _ctx) {
       version.user
     }
   end
 
-  field :annotation do
-    type -> { AnnotationType }
-
+  field :annotation, -> { AnnotationType }, 'Annotation associated with this log entry' do
     resolve ->(version, _args, _ctx) {
       version.annotation
     }
   end
 
-  connection :projects, -> { ProjectType.connection_type } do
+  connection :projects, -> { ProjectType.connection_type }, 'Projects associated with this log entry' do
     resolve ->(version, _args, _ctx) {
       version.projects
     }
   end
 
-  connection :teams, -> { TeamType.connection_type } do
+  connection :teams, -> { TeamType.connection_type }, 'Teams associated with this log entry' do
     resolve ->(version, _args, _ctx) {
       version.teams
     }
   end
 
-  field :task do
-    type -> { TaskType }
-
+  # TODO Don't we already have annotation above?
+  field :task, -> { TaskType } do # TODO Document
     resolve ->(version, _args, _ctx) {
       version.task
     }
   end
 
-  field :tag do
-    type -> { TagType }
-
+  # TODO Don't we already have annotation above?
+  field :tag, -> { TagType } do  # TODO Document
```

Comment:
Yes, this is only to return the specific annotation type - but we could convert those to a single field and use a `UnionType`